README.md

```diff
@@ -172,27 +188,73 @@ command.
 There are three views for reviewing a pull request on GitHub: **Conversation**,
 **Commits**, and **Files changed**.
 
-#### Conversation ####
+Conversation
+
```

Comment:
` ### Reviewing a pull request ###` was already a third level heading, and I wanted a bunch of subheadings - the way it was before, there was 3 subheadings that corresponded to the 

` There are three views for reviewing a pull request on GitHub: **Conversation**, **Commits**, and **Files changed**.`

so it seemed weird to have those three things that go together, followed by more fourth level headings about other things

does that make any sense? :p